### ISSUE TITLE
ffmpeg is not found by no means

### ISSUE THREAD

**Issue by Chis02** [scottpetrovic/animator-video-reference#3]
Hi! I found your plugin and thought it was great, just the tool I felt a lack of for animating on krita. But sadly, it doesn't want to work for me. I saw what you answered to the previous issue but nothing works for me. I have my ffmpeg folder on `C:\ffmpeg\bin`, I have it linked with the "render animation" option, I added it to my paths on windows 10 and restarted the pc twice... But the same "ffmepg not found" window pops up every time I try to use the plugin. It could be because of my krita version not being compatible with the plugin?? I work on krita 4.2.0
Sorry because I feel like you explained everything so easy and clearly, but I can't get it to work

**Comment by scottpetrovic** [comment]
For figuring things out, let's just take krita and plugins out of the equation.

Can you open up a command prompt (start menu and type "cmd"). Then type "ffmpeg" (without quotes). It should give you some information. If Windows says it doesn't recognize the command, it means that the environment variables aren't set up right.

**Comment by gsteinman** [comment]
Hi Scott,
I'm running Krita on OSX Catalina, and have installed FFmpeg using homebrew; typing FFmpeg in my command prompt yields information about the software. I still receive the notification that it - as well as FFprobe - cannot be found. Any thoughts?
Thanks

**Comment by scottpetrovic** [comment]
I don't know then. With you typing ffmpeg in the command prompt and it working, that is also all the plugin is doing. It just tries to grab the version number. If the command fails, the plugin says that ffmpeg isn't found. Maybe there is a difference between OSX and Linux and Windows. 

Either way, this ffmpeg linking seems to be a big issue for a lot of people. I am going to be later updating this plugin to just point to the ffmpeg location set in the render dialog. That way people won't have to mess with their environment variables.

**Comment by gsteinman** [comment]
Thanks for the reply!
That would be wonderful, please send an update once you've sent it out!
Cheers and good health

**Comment by TiMeBaNDiT76** [comment]
Yes, if you install ffmpeg on OSX, ffmpeg -version would be the command to get the version (instead of ffmpeg -v, which you used)
